content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -101,25 +101,27 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-### Latency for Istio 1.19
+### Latency for Istio 1.20
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
 proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
-the two proxies add about 1.31 ms and 1.58 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+the two proxies add about 0.228 ms and 0.298 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
 We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
-for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
+for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 2,4,8,16,32,64 client connections, 2 proxy workers and mutual TLS enabled.
 
```

Comment:
Can we add some documentation to the graphs or page to better differentiate the two graphs. I think a title would suffice. It's hard to tell just looking at the page that one is 90th percentile and one is 99th percentile without reading the test, etc. If you just look at the pictures, you see two with no titles, having the same horizontal and vertical axis, but different values. We should make it easier to understand the data just looking at the picture.